### PR TITLE
implemented getUsers endpoint

### PR DESCRIPTION
**Pull request by CoupDeWhoop**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -52,6 +52,22 @@ describe('GET requests', () => {
...
+                expect(body.users).toHaveLength(4)
+                body.users.forEach((user) => {
+                    expect(user).toHaveProperty("username", expect.any(String))
+                    expect(user).toHaveProperty("name", expect.any(String))
+                    expect(user).toHaveProperty("avatar_url", expect.any(String))
+                }) 
+            })
+        });
+    })
+
```
Comment (CoupDeWhoop) [inline __tests__/app.test.js:70]: Hi, I couldn't think of any errors to test for. I was thinking they either put in the right path or they don't

File: __tests__/app.test.js [hunk 2]
```diff
@@ -52,6 +52,22 @@ describe('GET requests', () => {
         })
     })
 
+    describe("GET /api/users", () => {
+        test.only('200 - should respond with an array of user objects', () => {
+            return request(app)
+            .get("/api/users")
+            .expect(200)
+            .then(({body}) => {
+                expect(body.users).toHaveLength(4)
```
Comment (Jodiech) [inline __tests__/app.test.js:61]: Nice to see you are checking the length here before the forEach 👍

File: controllers/users.controller.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const { fetchAllUsers } = require("../models/users.model")
+
+exports.getUsers = (req, res, next) => {
+    fetchAllUsers()
+    .then((users) => res.status(200).send({users}))
```
Comment (Jodiech) [inline controllers/users.controller.js:5]: Great that you are sending this back on a key of users 🤸

**Review by Jodiech — APPROVED** [review]
Hey 👋 nice work on this ticket! Happy for you to merge!

**Pull request merged**